### ISSUE TITLE
[a11y audit] Testing: Chatbot MHV Prescription Skill 4.2.16: Set expectations

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#67315]
## 4.2.16 Set expectations
- **Testing Status:** `Completed`
- **Findings**: `Present`
- **Audit Spreadsheet:** [Link to 4.2.16](https://docs.google.com/spreadsheets/d/12NgigOQTKgsSexnq_ioUwzXQLtRGRcrIYN--yb3i2Ig/edit#gid=1502697987)

### Testing Details
- **URL**: `dev`
- **Credentials**: user `#228`
- **OS**: MacOS 13.6
- **Browser**: Chrome, rolling release
- **a11y Audit Finding,** related to [4.2.16 Set expectations](https://mitre.github.io/chatbot-accessibility-playbook/docs/4_2_16.html)
- **Relates to:** n/a
- **VPAT Status:** Not evaluated
- **Impacts**: Cognitive
- **Experience Standards:**  `Category 03` `Issue 02`
- **Severity Rubric**: n/a
- **[Update reflected in Audit Spreadsheet](https://docs.google.com/spreadsheets/d/12NgigOQTKgsSexnq_ioUwzXQLtRGRcrIYN--yb3i2Ig/edit#gid=1502697987)** ✅

I tested for the following:

> Instruct the user that there are “no additional steps” if no more steps apply. Each conversation “endpoint” suggests additional actions the user can or should take.

### Does Not Pass 🔴 

When a user refills a prescription, there's no language explaining what will happen to that refill next. Will the user get email confirmation? Do they need to call their pharmacy to confirm? How long will it take for their prescription to be filled? Or is there **nothing** the user can do at this point, and can the copy reflect that?

### Recreating the Issue 🖥️ 

To view the issue, go to the prescription skill: https://dev.va.gov/contact-us/virtual-agent/

Go to the "Request refills" section, then refill a prescription. You'll see something like this:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/f53e5f2a-36df-4bcf-8cbc-9ce549bc4f0d)

### How to Fix 🛠️ 

Add copy to let the user what they can do next, if anything.

After the user has submitted all of their refills and the message "x refill requests submitted." comes up, you could say something like:
- "We'll send you a confirmation email." or
- "Call your pharmacist to confirm." or
- "If you have questions, call XXX." or
- "You have no more prescriptions to refill." or similar.

I'm not sure what the language should be, since I don't have the background to know what the user's options are here. But ideally, since the user took an action, it'd be good to give them someway to confirm it beyond the chatbot, someone to call, or just a "you're done!" confirmation.
